packages/prop-house-webapp/src/components/NoActiveHouseModal/NoActiveHouseModal.module.css

```diff
@@ -0,0 +1,106 @@
+.modal {
```

Comment:
Has this been used previously? If so, can we make it so that the base `Modal` component has these properties so as to not reuse code on every new `Modal` implementation. Better yet - can we refactor this to make a generalized `Modal` component?